docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -95,6 +95,7 @@ Alpha features might be changed or removed without prior notice.
 | `authnService`                     | Use new auth service to perform authentication                                                                                                                               |
 | `sessionRemoteCache`               | Enable using remote cache for user sessions                                                                                                                                  |
 | `alertingBacktesting`              | Rule backtesting API for alerting                                                                                                                                            |
+| `renderingOverJWT`                 | Uses JWT-based auth for rendering instead of relying on remote cache                                                                                                         |
```

Comment:
What does "Over" signify in the name of this toggle? I would have expected something like renderingAuthJWT or renderingJWTAuth

Reply:
Probably a non-native speaker mistake here, I'm happy to rename it!

JFYI, I thought "over" as "through", like "by using JWT", like when you say "HTTP over TLS". But again, happy to change it, any alternative that makes it more clear is completely fine for me.

Reply:
Thanks, yes I think it would be easier to understand the meaning if we change it to `renderAuthJWT`.

My reasoning is that in HTTP over TLS, TLS is used as a transport layer for HTTP, while here JWT is just an authentication method.

Reply:
Great!! Thanks for sharing your reasoning, that helps a lot to me as a non-native speaker! Really appreciated! 💟